src/Tv/Programme.php

```diff
@@ -62,14 +67,15 @@ class Programme implements XmlSerializable
     public $videoplus;
 
     /**
-     * @var string
+     * @var string Legal values are `0` or `1`.
      */
-    public $channel;
+    public $clumpidx;
 
     /**
-     * @var string Legal values are `0` or `1`.
+     * @var string A programme specific identifier for catchup URLs. Not part of the XMLTV specification.
+     * @see https://github.com/kodi-pvr/pvr.iptvsimple/tree/7.0.0-Matrix#supported-m3u-and-xmltv-elements
      */
-    public $clumpidx;
+    public $catchupId;
```

Comment:
:information_source: I didn't add it to constructor because it is not part of the official specification.

Reply:
ok, no problem ;)